src/MSBuild.Conversion.Project/Converter.cs

```diff
@@ -24,20 +25,18 @@ public Converter(UnconfiguredProject project, BaselineProject sdkBaselineProject
             _differs = GetDiffers();
         }
 
-        public void Convert(string outputPath, string? defaultTFM, bool keepCurrentTfm)
+        public void Convert(string outputPath, string? specifiedTFM, bool keepCurrentTfm, bool usePreviewSDK)
         {
-            ConvertProjectFile(defaultTFM, keepCurrentTfm);
-            CleanUpProjectFile(outputPath);
+            var result = ConvertProjectFile(specifiedTFM, keepCurrentTfm, usePreviewSDK);
+            if (result is { })
```

Comment:
I have so much the giant baby

Reply:
please just use `result != null` like a normal person

---
In reply to: [422375425](https://github.com/dotnet/try-convert/pull/291#discussion_r422375425) [](ancestors = 422375425)

Reply:
actually I can just get rid of this nonsense, it's an artifact from something else I did but ended up not doing